wrappers/android/examples/native_example/app/src/main/cpp/native-lib.cpp

```diff
@@ -9,5 +16,43 @@ Java_com_example_realsense_1native_1example_MainActivity_nGetLibrealsenseVersion
 extern "C" JNIEXPORT jint JNICALL
 Java_com_example_realsense_1native_1example_MainActivity_nGetCamerasCountFromJNI(JNIEnv *env, jclass type) {
     rs2::context ctx;
-    return ctx.query_devices().size();;
-}
\ No newline at end of file
+    return ctx.query_devices().size();
+}
+
+extern "C" JNIEXPORT jint JNICALL
+Java_com_example_realsense_1native_1example_MainActivity_nGetSensorsCountFromJNI(JNIEnv *env, jclass type) {
+    rs2::context ctx;
+    return ctx.query_all_sensors().size();
+}
+
+rs2::pipeline pipe;
+rs2::config cfg;
```

Comment:
`static`?

Reply:
good question. Android loads native libraries once per process, so these variables are effectively static. Should I still declare them as static to comply with code style guidelines?

Reply:
I suggest putting `static` in front of both.  These variables are only used in this one file, so it's appropriate.  Especially given their generic names.